tests/vault/test_vault_client_context.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Test vault
+  hosts: ipaserver
```

Comment:
Do we also need `hosts: ipaclients, ipaserver` here?

Reply:
No, as we only need to test the failure in the case where `ipaapi_context: server`.